Ore tendril pattern with slight impurities

BlueRock2 opened the pull request:
Instead of the heavily mixed ore pattern currently used with Diggy, this alternative ore pattern contains ore tendrils that are about 80% pure ore with lower impurities.   Feedback encouraged.

BlueRock2 committed:
Ore tendril pattern with slight impurities

Instead of the heavily mixed ore pattern used with Diggy in the past, this ore pattern contains ore tendrils that are about 80% pure ore with lower impurities.   Feedback encouraged.

linaori reviewed (approved):
I can't see anything wrong with this, as it's not enabled by default, it looks safe to merge

BlueRock2 committed:
Update tendrils_impure.lua

Added in a small amount of scattered ore since it may take players a little time to find each of the four primary ore types.  Slight reduction in copper and iron ore.

plague006 commented:
Looks good in testing. I think you should add it as an option on line 302 of diggy config

plague006 commented:
![image](https://user-images.githubusercontent.com/10162554/49696866-7ad48800-fb7e-11e8-8004-9fdefeb19ecd.png)

linaori reviewed (approved):
I'm still afraid the tendrils are too big, but it looks really good like this already
